File: Tutorial_Sample_Code/DataConnectivity/DataSets/VB/VB_Win_Data_DataSets/Form1.vb
Imports CrystalDecisions.CrystalReports.Engine
Imports CrystalDecisions.Shared
Imports System.Data


Public Class Form1

    Private customerReport As ReportDocument

    Private Sub ConfigureCrystalReports()
        customerReport = New ReportDocument()
        Dim reportPath As String = Application.StartupPath & "\" & "Customer.rpt"
        customerReport.Load(reportPath)
        Dim myDataSet As DataSet = DataSetConfiguration.CustomerDataSet
        customerReport.SetDataSource(myDataSet)
        myCrystalReportViewer.ReportSource = customerReport


    End Sub

    Private Sub Form1_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
        ConfigureCrystalReports()
    End Sub
End Class

File: Tutorial_Sample_Code/DataConnectivity/DataSets/CS/CS_Win_Data_DataSets/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using CrystalDecisions.CrystalReports.Engine;
using CrystalDecisions.Shared;
using CS_Win_Data_DataSets;

namespace CS_Base
{
    public partial class Form1 : Form
    {
        private ReportDocument customerReport;

        public Form1()
        {
            InitializeComponent();
        }

        private void ConfigureCrystalReports()
        {
            customerReport = new ReportDocument();
            string reportPath = Application.StartupPath + "\\" + "Customer.rpt";
            customerReport.Load(reportPath);
            DataSet dataSet = DataSetConfiguration.CustomerDataSet;
            customerReport.SetDataSource(dataSet);
            crystalReportViewer.ReportSource = customerReport;
        }

        private void Form1_Load(object sender, EventArgs e)
        {
            ConfigureCrystalReports();

        }

        private void crystalReportViewer_Load(object sender, EventArgs e)
        {

        }
    }
}